TodoListApp/src/TodoListApp/Models/TodoItemTable.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace TodoListApp.Models
+{
+    public class TodoItemTable
```

Comment:
- `.Models` isn't a proper namespace for db data - move to `.Data`
- `TodoItemTable` doesn't sound well tho, rename to `TodoItem`